dist.ini

```diff
@@ -1,13 +1,19 @@
-name = File-Find
+name = File::Find
```

Comment:
What is this used for? On one hand, the name of the released distribution is already `File::Find` but on the other hand, the folder of the repo better stays `File-Find` in my opinion.

Reply:
That is for 'mi6' use. I think it will work either way, but the "real" module package name is best.

Reply:
Fair enough. Seems like it's used for figuring out the "main module" so why not.